client/lib/form-state/store/core.js

```diff
@@ -4,7 +4,9 @@
  * Internal dependencies
  */
 
-import { changeFieldValue } from '../';
+import formState from '../';
+
+const { changeFieldValue } = formState;
```

Comment:
Should we add a comment for why we need to destructure outside of the import?

Reply:
I'm going to propose "no" because we've done it quite a few times already. If someone tries to change it after we stop compiling to CommonJS then webpack will immediately stop them from doing it.

Reply:
Agreed. I think for a lot of people there will be surprises with the new webpack behavior but it will quickly result in learning + better code